docs/projects/ansible/collection/centurion/playbooks/teams.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Teams
+description: No Fuss Computings Companion Ansible Collection Centurion teams Playbook.
+date: 2024-08-18
+template: project.html
+about: https://gitlab.com/nofusscomputing/projects/ansible/collections/kubernetes
+---
+
+The teams playbook has been created for the purpose of creating standardised teams with a defined set of permissions. With this playbook it is possible to create every team within an organisation and define set teams for newly created team. The playbook can also be used to patch a team with updated permissions or notes.
+
+The teams playbook includes the [AWX Feature](../../../playbooks/awx.md) to import the playbook as a job template in AWX / Ansible Automation Platform.
+
+The following job template will be created:
+
+*  **Centurion/Access/Teams** Creation and patching of teams and permissions
+
+On import to AWX / Ansible Automation Platform a credential type will also be created, `Playbook/teams/centurion` that can be used to supply the required secrets and Centurion host.
+
+!!! info Centurion ERP API
+    The playbook is designed to use the Centurion ERP API to fetch, create and patch teams and permissions.
+
+
+## Play workflow
+
+The teams playbook gathers information regarding centurion organisations from the ansible inventory and from Centurion itself. using this information the play is designed to create new teams, patch permissions and patch notes. The workflow for the playbook is as follows
+
+- Fetch all organisations
+- Fetch all existing teams within each organisation
+- Fetch any teams to be created from inventory
+- Create any new teams that do not exist in centurion
+- Fetch all teams from all organisations in Centurion
+- Patch all teams with required permissions
+- Patch all teams with required notes
+
+
+## Configuration
+
+The teams playbook uses variables that are gathered from inventory
+The expected structure of the inventory file is:
+
+```yaml
+centurion_erp:
+  teams:
+
+    - name: "organisation name"
+      teams:
+      - name: "team-name"
+        permissions: []
+        notes: "permissions must be a list"
```

Comment:
tidy this up. spacing or no spacing, choose the one easier to read

Reply:
Done, went with no spacing